css/css-text/letter-spacing/letter-spacing-bidi-003.xht

```diff
@@ -0,0 +1,48 @@
+﻿<html xmlns="http://www.w3.org/1999/xhtml">
+  <head>
+    <meta charset="utf-8"/>
+    <title>CSS Text level 3 Test: letter spacing, justified text and bidi</title>
+    <link rel="author" title="Mike Bremford" href="http://bfo.com" />
+    <link rel="help" href="https://drafts.csswg.org/css-text-3/#letter-spacing-property" />
+    <link rel="help" href="https://drafts.csswg.org/css-writing-modes-4/#bidi-fragmentation" />
+    <link rel="match" href="reference/letter-spacing-bidi-003-ref.xht" />
+    <meta name="assert" content="text-align: justify will stretch the space between words, which should be applied after bidi processing."/>
+    <style>
+      .test, .control {
+          font: 50px/1 monospace;
+          margin: 0;
+          width: 14ch;
+          font-kerning: none;
```

Comment:
Thanks Florian, good observations all of them. Most applied as suggested, but for this one:

* width matters in letter-spacing-003 as it's required to ensure the justification matches the letter spacing, but not in the others;
* monospace matters as we need to ensure the space character is the same width as the other characters - the line-height is simply there to keep things consistent across browsers - it's not strictly necessary I suppose given we have a reference rendering, but it makes comparison between rendering engines a bit easier. 
* Same argument for font-kerning, although it's unlikely kerning exists between any of these letters, we can make no guarantees about the font: if it did it would break the references tests.
* margin matters in none of them; removed

Reply:
Sorry, I only meant to comment about font-kerning in this comment (margin was in a previous on). It's github who decided to show more lines of context and make it look like I might have been complaining about the rest too.

As for Kerning, you're right. `letter-spacing` is in addition to kerning, not instead of it, so if there was kerning, it would break without `font-kerning: none;`.